code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -29,7 +29,7 @@ client/proc/one_click_antag()
   // You pass in ROLE define (optional), the applicant, and the gamemode, and it will return true / false depending on whether the applicant qualify for the candidacy in question
 	if(jobban_isbanned(M, "Syndicate"))
 		return FALSE
-	if(M.stat || !M.mind || M.mind.special_role)
+	if(M.stat || !M.mind || M.mind.special_role || M.mind.offstation_role)

```

Comment:
This seems somewhat fragile of a fix - are you sure there aren't other places where antags are created, such as autotraitor?

Reply:
I think this is the best solution, as folks with offstation roles should never be selected for antagonist positions. I'll have a look at other antagonist selections.